test/e2e/storage/framework/testdriver.go

```diff
@@ -205,6 +206,8 @@ type DriverInfo struct {
 	StressTestOptions *StressTestOptions
 	// [Optional] Scale parameters for volume snapshot stress tests.
 	VolumeSnapshotStressTestOptions *VolumeSnapshotStressTestOptions
+	// [Optional] Parameters for performance tests
+	PerformanceTestOptions *PerformanceTestOptions
```

Comment:
If someone wants to run stress tests with different parameters (for example, different storage classes or different sizes), then this can only be done by defining different test drivers.

I think that's okay, I just wanted to call this out. The alternative (multiple `PerformanceTestOptions` and then instantiating the test multiple times) would be complex, too.